src/rules/props-order.ts

```diff
@@ -41,11 +42,11 @@ export const propsOrderRule: TSESLint.RuleModule<"invalidOrder", []> = {
           return aPriority - bPriority;
         });
 
+        const sourceCode = getSourceCode();
         for (const [index, attribute] of node.attributes.entries()) {
           if (attribute.type !== AST_NODE_TYPES.JSXAttribute) {
-            return;
+            continue;
```

Comment:
If `attribute` and `sortedAttribute` are both `spreadAttributes`, this logic will miss the error.
But given this rareness, I think we can ignore it for now.